lib/workos/organizations.rb

```diff
@@ -180,6 +180,32 @@ def delete_organization(id:)
         response.is_a? Net::HTTPSuccess
       end
 
+      # Retrieve a list of roles for the given organization.
+      #
+      # @param [String] organizationId The ID of the organization to fetch roles for.
+      def list_organization_roles(organization_id:)
+        response = execute_request(
+          request: get_request(
+            path: "/organizations/#{organization_id}/roles",
+            auth: true,
+          ),
+        )
+
+        parsed_response = JSON.parse(response.body)
+
+        roles = parsed_response['data'].map do |role|
+          ::WorkOS::Role.new(role.to_json)
+        end
+
+        WorkOS::Types::ListStruct.new(
+          data: roles,
+          list_metadata: {
+            after: nil,
+            before: nil,
+          },
+        )
```

Comment:
This is a bit silly. There's no pagination for this API, so perhaps I should just create a new type. Let me know your thoughts.

Reply:
I suppose you could just return the `roles` array directly if you don't think it'll ever be a paginated API.

But if there's a chance it'll become paginated later, then returning this at least prevents the need to introduce a breaking change later.

Reply:
Having to introduce a breaking change later is my worry. Pagination doesn't really make sense right now from a product perspective, but I worry if this will change as we see more use cases. I'll leave as-is for now.